mods/ra/rules/defaults.yaml

```diff
@@ -954,6 +952,8 @@
 	RevealOnDeath:
 		Duration: 60
 		Radius: 4c0
+	CustomOwnerLostAction:
+		Action: Nothing
```

Comment:
IMO this should be ChangeOwner to Neutral and then you should add a KeepEffectiveOwner boolean along.

Reply:
It would still get, mostly graphical after that point, issues like getting bounty text for neutral player and contrails being white.

Tho i see no problem with making effective owner an option for the trait.